worker/src/lib/HeliconeProxyRequest/forwarder.ts

```diff
@@ -61,6 +63,61 @@ export async function proxyForwarder(
     }
   }
 
+  if (
+    request.headers.get("Helicone-Prompt-Security-Enabled") &&
+    provider === "OPENAI"
+  ) {
+    let latestMessage;
+
+    try {
+      latestMessage = JSON.parse(request["cachedText"] ?? "").messages.pop();
```

Comment:
what is cachedText?

Reply:
@chitalian , changed to bodyText from proxyRequest